### ISSUE TITLE
openssl_decrypt() fails randomly when images are uploaded 

### ISSUE THREAD

**Issue by alinmiron** [GrafiteInc/CMS#133]
steps to reproduce: cms/images/create. click on "Drop files here to upload". Choose image, confirm.  Press Save.
 ErrorException (E_WARNING)
openssl_decrypt(): IV passed is only 6 bytes long, cipher expects an IV of precisely 16 bytes, padding with \0
see:
![screenshot-2018-4-17 whoops there was an error](https://user-images.githubusercontent.com/9200637/38871318-2c2986d0-4259-11e8-96c0-3bca33386227.png)

**Comment by mlantz** [comment]
Hey @alinmiron I have not been able to replicate this, what platform are you working on? Homestead?

**Comment by alinmiron** [comment]
Hi, 
I'm using Ubuntu 16.04 LTS 64-bit, PHP 7.1.16, MySQL 5.7 and Laravel development server localhost:8000 (php artisan serve)

**Comment by mlantz** [comment]
I believe this could be an issue of missing the encryption key, I have never been able to replicate the issue.

**Comment by mattkatt** [comment]
I am encountering this error today: it only occurs on a small number of images - attempting to upload different images before and after does not cause the issue, unless the new image itself ALSO explicitly throws the error.

We've looked into this and found that, after attempting to upload a broken file, the form response from cms/image is returning with location[undefined] in the form data form size, mime, original and name, suggesting an issue with the JS not saving the file correctly. I've included an image that is causing this error repeatedly:

![istock-501819453](https://user-images.githubusercontent.com/37899356/51745108-c7761300-2099-11e9-964e-ced55977e4e0.jpg)

This image has caused the same error to appear on multiple machines here. Note this is with internal storage and NOT s3.
